src/lib/components/custom/theme-selector.svelte

```diff
@@ -3,21 +3,12 @@
 	import Radio from '../ui/radio-button/radio.svelte'
 	import { ColorPalette } from 'carbon-icons-svelte'
 	import { theme } from '$lib/stores/theme.svelte'
-	import { calculateLuminance } from '@waku-objects/luminance'
-	import { getEffectiveColorMode } from '$lib/utils/colors'
 	import RadioGroup from '../ui/radio-button/radio-group.svelte'
+	import Button from '../ui/button.svelte'
 
-	let effectiveMode = $derived(getEffectiveColorMode(theme.mode))
-
-	let fill = $derived(
-		calculateLuminance(theme.baseColor) > 0.5
-			? effectiveMode === 'dark'
-				? 'fill: var(--colors-base)'
-				: 'fill: var(--colors-top)'
-			: effectiveMode === 'dark'
-				? 'fill: var(--colors-top)'
-				: 'fill: var(--colors-base)',
-	)
+	// without this line the dropdown does not appear...
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	const mode = $derived(theme.mode)
```

Comment:
This is indeed very weird. Please make an issue and I'd be happy to figure out what is the problem.

Reply:
Thanks, I opened an issue about it:
https://github.com/diete-design/diete.design/issues/139

Reply:
About your first point, there was an issue in the design file, for some reason it was not using the actual color picker component. It's fixed now.